localstack/services/apigateway/invocations.py

```diff
@@ -1,15 +1,10 @@
 import json
```

Comment:
note: mostly cleanups / refactoring in this file (migrating logic into `integrations.py`), practically no functional changes.